src/test/java/uk/gov/ons/ssdc/caseprocessor/schedule/WaveOfContactIT.java

```diff
@@ -30,17 +31,21 @@
 import uk.gov.ons.ssdc.caseprocessor.model.repository.EventRepository;
 import uk.gov.ons.ssdc.caseprocessor.model.repository.UacQidLinkRepository;
 import uk.gov.ons.ssdc.caseprocessor.model.repository.WaveOfContactRepository;
+import uk.gov.ons.ssdc.caseprocessor.testutils.QueueSpy;
+import uk.gov.ons.ssdc.caseprocessor.testutils.RabbitQueueHelper;
 import uk.gov.ons.ssdc.caseprocessor.utils.ObjectMapperFactory;
-import uk.gov.ons.ssdc.caseprocessor.utils.QueueSpy;
-import uk.gov.ons.ssdc.caseprocessor.utils.RabbitQueueHelper;
 
 @ContextConfiguration
 @SpringBootTest
 @ActiveProfiles("test")
 @RunWith(SpringJUnit4ClassRunner.class)
 public class WaveOfContactIT {
-  private static final String OUTBOUND_PRINTER_QUEUE = "Action.Printer";
-  private static final String UAC_QUEUE = "case.rh.uac";
+  @Value("${queueconfig.outbound-printer-queue}")
```

Comment:
this already exists in `application.yml` and it's called `queueconfig.print-queue`